requirements_test.txt

```diff
@@ -21,11 +21,13 @@ opencv-python
 sortedcontainers==2.4.0
 h5py==3.14.0
 multiprocess>=0.70.12
+umap-learn==0.5.7
 
 # frameworks
 
 tensorflow==2.18.1
 keras==3.10.0
+tensorflow-addons>=0.13.0
```

Comment:
Can we avoid `tensorflow-addons` and replace it with `tensorflow`? We have included refactoring for 1.20 to remove it from ART by replacing all functions from `tensorflow-addons`. It started to affect upgrading to `tensorflow>=2.16.0` versions.